src/styles/main.scss

```diff
@@ -1,3 +1,68 @@
+@import './variables';
+
+* {
+  margin: 0;
+}
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  padding: 10px;
+  border: 2px solid red;
+  width: $size-cell * 7 + 6;
```

Comment:
it's better to use variables you have for width. You have `$column` for `7` and for `6` you can make a calculation as well using `$column`